packages/file/lib/src/interface/error_codes.dart

```diff
@@ -225,361 +225,361 @@ class _LinuxCodes implements _Codes {
   const _LinuxCodes();
 
   @override
-  final int e2big = 7;
+  int get e2big => 7;
```

Comment:
Is this not valid the other way?

Reply:
It triggers a lint about avoiding const final fields on final classes because it wastes the memory space of a field.